internal/kubeconfig/contexts.go

```diff
@@ -54,8 +54,19 @@ func (k *Kubeconfig) ContextNames() []string {
 	}
 
 	var ctxNames []string
+ctxLoop:
```

Comment:
I believe this could be refactored to avoid syntax that feels like a ` goto` to me, maybe having the filtring as a separate function. I'm open to suggestions